docker-compose.yml

```diff
@@ -16,6 +16,22 @@ services:
     depends_on:
       - localstack
 
+  awslocal:
+    build:
+      context: scripts
+      dockerfile: Dockerfile-awslocal
```

Comment:
This is a simple dockerized version of the `awslocal` CLI. If we like this pattern, we could consider publishing a built image of this for re-use across repositories.